pkg/minikube/cruntime/cruntime.go

```diff
@@ -78,6 +79,8 @@ type Config struct {
 	Socket string
 	// Runner is the CommandRunner object to execute commands with
 	Runner CommandRunner
+	// Kubernetes config
+	KubernetesConfig config.KubernetesConfig
```

Comment:
Rather than making `cruntime` aware of the entire `KubernetesConfig` object, could you please add fields for `ImageRepository` and `KubernetesVersion` to `cruntime.Config`? This will reduce the scope and make future refactors easier.

Context: https://zh.wikipedia.org/wiki/%E5%BE%97%E5%A2%A8%E5%BF%92%E8%80%B3%E5%AE%9A%E5%BE%8B

Reply:
Done